File: Programaciòn Orientada a Objetos/Algoritmo de Ordenamiento Burbujeo/Burbujeo.java
import java.util.concurrent.ThreadLocalRandom;

public class Burbujeo{
    public static int[] burbujeo(int vector[]){
         for(int i = 0; i < vector.length-1; i++){
            for(int j = i + 1; j < vector.length; j++){
                if(vector[i] > vector[j]){
                    int auxiliary = vector[i];
                    vector[i] = vector[j];
                    vector[j] = auxiliary;
                }
            }
         }
        
        return vector;
    }
    
    public static void main(String[] args){
        final int SIZE = 10000;
        
        int vector[] = new int[SIZE];
        
        for(int i = 0; i < SIZE; i++){
            vector[i] = ThreadLocalRandom.current() .nextInt(0, SIZE + 1);
        }
        
        vector = burbujeo(vector);
        
        for(int i = 0; i < SIZE; i++){
            System.out.println(vector[i]);
        }
    }
}

File: Programaciòn Orientada a Objetos/Algoritmo de Ordenamiento Burbujeo/burbujeo.py
from random import randint

SIZE=10000

vector = []

def burbujeo(vector):
    for i in range(0, SIZE - 1):
        for j in range(i + 1, SIZE):
            if vector[i] > vector[j]:
                auxiliary = vector[i]
                vector[i] = vector[j]
                vector[j] = auxiliary
                
for i in range(SIZE):
    vector.append(randint(0, SIZE))
     
burbujeo(vector)

for i in range(SIZE):
	print (vector[i])
